blog/2023-11-27-magazine-006.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Inflight Magazine no. 6
+description: Updates from the winglang project
+authors: 
+  - revitalb
+  - urib
+tags: [cloud-oriented programming, winglang, changelog, newsletter,]
+hide_table_of_contents: true
+---
+> The 6th issue of the Wing Inflight Magazine.
+> <!--truncate-->
+
+Hey folks!
+
+We’re back with another issue of the **Wing Inflight Magazine** and we are excited to share some updates about [winglang](https://winglang.io).
+
+- 🚀 [New: bring ui](#bring-ui)
+- 🚀 [New: Singleton](#singleton)
+- 🔥 [Remove $ from String Interpolations](#remove--from-string-interpolations)
+- 🚀 [New: Expose AWS Resources](#expose-aws-resources)
+- 🚀 [New: Sourcemaps and Stacktraces](#sourcemaps-and-stacktraces)
+- 📅 [Upcoming Events](#upcoming-events)
+- 🎥 [The Wingly Update](#the-wingly-update)
+
+### bring ui
+
+Introducing the ui module to the SDK, a collection of APIs for adding visual interactions for your classes in the Wing Console. UI components aren't displayed in the Wing Console quite yet, but they can be accessed and called through the simulator API!
```

Comment:
I will probably regret it when we send the email, but maybe add a picture of the UI that got created?

Reply:
Removed this topic per Elad's request.